manifests/2.1.0/opensearch-2.1.0.yml

```diff
@@ -40,7 +40,7 @@ components:
       - gradle:dependencies:opensearch.version
   - name: security
     repository: https://github.com/opensearch-project/security.git
-    ref: main
+    ref: '2.x'
```

Comment:
Hey @DarshitChanpura since we are about to create the release candidates, can we cut 2.1 branch?
Thanks!

Reply:
+1 2.1